package.json

```diff
@@ -60,12 +63,15 @@
     "babel-jest": "28.1.1",
     "babel-plugin-istanbul": "6.1.1",
     "babel-plugin-transform-imports": "^2.0.0",
-    "cypress": "9.6.1",
+    "cors": "2.8.5",
+    "cypress": "^10.3.0",
     "eslint": "8.17.0",
     "eslint-loader": "4.0.2",
     "eslint-plugin-cypress": "2.12.1",
+    "express": "4.18.1",
     "identity-obj-proxy": "3.0.0",
     "jest": "28.1.1",
+    "lodash": "4.17.21",
```

Comment:
Let's actually add `lodash` to dependencies as it is used inside components and has until now been only a transitive dependency of other packages.